bundles/org.openhab.binding.velux/src/main/java/org/openhab/binding/velux/internal/bridge/slip/io/DataInputStreamWithTimeout.java

```diff
@@ -75,34 +88,47 @@ public void interrupt() {
          */
         @Override
         public Boolean call() throws Exception {
+            logger.trace("Poller.call(): started");
```

Comment:
I'm a bit in doubt about these log statements in the light of https://www.openhab.org/docs/developer/guidelines.html#f-logging

If there are no other complaints, I would say we should merge it

Reply:
Hi @martinvw I do understand that the developer guideline says one should not use the logger to replace the debugger. But our problem is that the Velux bridge has an undocumented weakness whereby it sometimes (very infrequently) falls into a "zombie state" that requires the bridge to be physically power cycled. Users of the binding have been trying to track down the exact circumstances when such zombie states occur, but it is very infrequent, so I can't expect normal users to be trying to run the binding in a debugger. That is the reason why I added these `logger.trace()` commands instead. So I do hope you that can allow this please.